chapter_3/ex3_floating.rb

```diff
@@ -0,0 +1,24 @@
+#!/usr/bin/evn ruby
+puts 'I will now count my chickens:'
+
+puts "Hens #{25.0 + 30.0 /6.0}"
```

Comment:
FYI... If any of the operands are floating point, it will switch to floating point math.  So `25 + 30 / 6.0` is equivalent.